python/parquet_tools/common/cmdinfo.py

```diff
@@ -0,0 +1,80 @@
+# *****************************************************************************
+# *                            c m d i n f o . p y                            *
+# *****************************************************************************
+#
+# This file is part of parquet_tools.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""This module contains an argparse wrapper class that handles common
+options for parquet_tools commands.
+"""
+
+__all__ = ['CmdInfo']
+
+from parquet_tools.common.errhandlers import ErrInfo, ParmError
+
+
+# *****************************************************************************
+# *                               C m d I n f o                               *
+# *****************************************************************************
+
+class CmdInfo:
+    """The CmdInfo class processes the command syntax and parses the
+    command line using that syntax.
+    """
+
+    # Command specification
+    #
+    opt = None        # Instance of the argparse object
+
+    # *************************************************************************
+    # *                   p a r s e _ c o m m a n d l i n e                   *
+    # *************************************************************************
+    @classmethod
+    def parse_commandline(cls, apobj):
+        """Parse the command line using a supplied command definition.
+
+        Parameters
+        ----------
+        apobj : 'argprase object'
```

Comment:
normally there would be a description here of what the object is for or does.
I think the object type does not need to be in quotes.

Reply:
Correct, I am missing argument documentation. That I will add. As for quotes, I was strictly following he examples in https://developer.lsst.io/python/numpydoc.html#py-docstring-class-structure where type type is always quoted. So, unless I am misreading the example, the quotes will remain.